src/functionalTest/resources/responses/response-caseworker-postpone-hearing-about-to-submit.json

```diff
@@ -0,0 +1,45 @@
+{
+  "data": {
+    "cicCaseHearingList": {
+      "value": {
+        "code": "18b88d33-9da8-4ee0-946d-de71d33e4d6f",
+        "label": "1 - Case management - 21 Apr 2023 09:00"
+      },
+      "list_items": [
+        {
+          "code": "18b88d33-9da8-4ee0-946d-de71d33e4d6f",
+          "label": "1 - Case management - 21 Apr 2023 09:00"
+        }
+      ],
+      "valueLabel": "1 - Case management - 21 Apr 2023 09:00",
+      "valueCode": "18b88d33-9da8-4ee0-946d-de71d33e4d6f"
+    },
+    "cicCaseHearingNotificationParties": [],
+    "cicCaseRespondentName": "Appeals team",
+    "cicCaseRespondentOrganisation": "CICA",
+    "cicCaseRespondentEmail": "appeals.team@cica.gov.uk",
+    "cicCaseFirstDueDate": "",
+    "stayIsCaseStayed": "No",
+    "hearingStatus": "Postponed",
+    "hearingType": "CaseManagement",
+    "date": "2023-04-21",
+    "hearingTime": "09:00",
+    "postponeAdditionalInformation": "test",
+    "hearingList": [
+      {
+        "id": "1",
+        "value": {
+          "hearingStatus": "Postponed",
+          "hearingType": "CaseManagement",
+          "date": "2023-04-21",
+          "hearingTime": "09:00"
+        }
+      }
+    ],
+    "hyphenatedCaseRef": "1234-5678-9012-3456",
+    "closedDayCount": "",
```

Comment:
You can remove all null and empty string values as they are unhelpful and tell us nothing. you are already using the following in your json comparison so any unhelpful values can be removed.
            .when(IGNORING_EXTRA_FIELDS)

Reply:
Actioning now